ui/app/templates/vault/cluster/unseal.hbs

```diff
@@ -31,21 +31,29 @@
           is
           {{if this.model.unsealed "unsealed" "sealed"}}
         </p>
-        <p>
-          Unseal Vault by entering portions of the unseal key. This can be done via multiple mechanisms on multiple
-          computers. Once all portions are entered, the root key will be decrypted and Vault will unseal.
-        </p>
+        {{#if this.model.unsealed}}
+          <p>Please wait while we redirect you.</p>
+        {{else}}
+          <p>
+            Unseal Vault by entering portions of the unseal key. This can be done via multiple mechanisms on multiple
+            computers. Once all portions are entered, the root key will be decrypted and Vault will unseal.
+          </p>
+        {{/if}}
+        {{#unless this.model.unsealed}}
```

Comment:
could this unless statement be removed and this component moved inside the `{{else}}` block above?